blog/2023-10-24-monitoring-vs-observability/index.md

```diff
@@ -0,0 +1,138 @@
+
+##### Logs: Event-Driven in Monitoring vs Queryable in Observability
+
+Logs provide a detailed account of events and are fundamental to both monitoring and observability. However, the treatment differs.
+
+Monitoring: Event-Driven Logs
+In monitoring systems, logs are often used for event-driven alerting. For instance, a log entry indicating a failed login attempt might trigger an alert for potential security concerns. These logs are essential but are typically consulted only when an issue has already been flagged by the monitoring system.
+
+Observability: Queryable Logs
+In observability platforms, logs are not just passive records; they are queryable data points that can be integrated with metrics and traces for a fuller picture of system behavior. You can dynamically query logs to investigate anomalies in real-time, correlating them with other high-cardinality data to understand the 'why' behind an issue.
+
+#### Proactive vs Reactive
+
+The second key difference lies in how these approaches are generally used to interact with the system.
+
+##### Monitoring: Set Alerts and React
+
+Monitoring is generally reactive. You set up alerts for known issues, and when those alerts go off, you react. It’s like having a fire alarm; it will notify you when there’s a fire, but it won’t tell you how the fire started, or how to prevent it in the future.
+
+##### Observability: Continuous Exploration
+
+Observability, by contrast, is more proactive. With an observability platform, you’re not just waiting for things to break. You’re continually exploring your data to understand how your system behaves under different conditions. This allows for more preventive measures and enables engineers to understand the system’s behavior deeply.
+
+#### Opinionated Dashboards and Charts
+
+Navigating the sprawling landscape of system data can be a daunting task, particularly as systems scale and evolve. Both monitoring and observability tools offer dashboards and charts as a solution to this challenge, but the philosophy and functionality behind them can differ significantly.
+
+##### Monitoring: Pre-Built and Prescriptive Dashboards
+In the realm of monitoring, dashboards are often pre-built and prescriptive, designed to highlight key performance indicators (KPIs) and metrics that are generally considered important for the majority of use-cases. For instance, a pre-configured dashboard for a database might focus on query performance, CPU usage, and memory consumption. These dashboards serve as a quick way to gauge the health of specific components within your system.
+
+Advantages:
```

Comment:
this entire bullet/points section needs to be reviewed, I'm not even sure how you wanted to structure it @shyamvalsan  😅